source/catalog.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Каталог Седона</title>
+  </head>
+  <body class="page_ _body">
+    <header class="main-header">
+      <nav class="main-nav">
+      <div class="main-nav_ _wrapper">
+        <ul class="main-nav_ _list site-list">
+        <li class="site-list_ _item site-list_ _item-current "><a href="#">ГЛАВНАЯ</a>
+          <li class="site-list_ _item"><a href="#">ФОТО И ВИДЕО</a>
+            <li class="site-list_ _item"><a href="#"><img class="main-nav_ _link_ _logo" src="img/logotype-phone.svg" width="185" height="96" alt="Логотип Седона."></a>
+              <li class="site-list_ _item"><a href="#">ФОРМА ОТЗЫВА</a>
+                <li class="site-list_ _item"><a href="#">HTML ACADEMY</a>
+        </ul>
+      </div>
+      </nav>
+    </header>
+    <main class="main-container">
+      <section class="banner banner--advertisement ">
+        <h1 class="visually-hidden">WELCOM to the GORGEOUS SEDONA</h1>
+      </section>
+      <section class="benefits features">
+        <h2 class="features_ _title headline">ФОТО И ВИДЕО</h2>
+        <p class="features_ _description benefits_ _description">Не можете решиться на путешествие из-за курса? Фотографии помогут вам забыть о политике и экономике.</p>
+      <ul class="features_ _list">
+      <li class="features_ _item">
+        <img class="features_ _image" src="./img/catalogfoto1.jpg" width="1200" height="400" alt="Фото дороги.">
+      <div class="features_ _image_ _info features_ _image_ _info--inside">
```

Comment:
Тут тоже надо прпоавить Блок. Так же и в аналогичных местах
```suggestion
      <div class="features__image-info features__image-info--inside">
```